conf/script/ha_setup

```diff
@@ -18,14 +18,9 @@
 HA_PATH="<HA_PATH>"
 SOURCE_HA_CONF="<HA_PATH>/conf/etc"
 HA_CONF="/etc/cortx/ha"
+LOGROTATE_CONF="/etc/logrotate_hourly.d"
```

Comment:
It should be named as LOGROTATE_CONF_DIR. This is a directory, not a config file.

Reply:
Done